bpf/lib/dbg.h

```diff
@@ -138,9 +138,18 @@ enum {
 #include "common.h"
 #include "utils.h"
 
+/* This takes both literals and modifiers, e.g.,
+ * printk("hello\n");
+ * printk("%d\n", ret);
+ *
+ * Two caveats when using this:
+ * - message needs to end with newline
+ * - only a subset of specifier are supported:
```

Comment:
Adding this comment is an excellent idea.

I'd add a third item to the list: You cannot use more than 3 format specifiers in the format string (This is because BPF helpers take a maximum of 5 arguments, and we already have format string and its size so we can only pass up to three more arguments to print).

Reply:
Done :) 
- Removed printt.
- Added the third constraint in comment.